docs/source/trainer/algorithms.rst

```diff
@@ -167,10 +167,12 @@ training loop at various :class:`.Events` and effect their changes by modifing t
 .. state.model = model()
 .. state.train_dataloader = train_dataloader()
 .. state.optimizers = optimizers()
+.. load_checkpoint()
```

Comment:
Is there a reason this whole block is commented out but is still here?

Reply:
CC: @growlix